tests/wpt/meta/html/dom/reflection-tabular.html.ini

```diff
@@ -7169,9 +7142,6 @@
   [th.colSpan: setAttribute() to 4294967296]
     expected: FAIL
 
-  [th.colSpan: setAttribute() to 1001]
-    expected: FAIL
-
   [th.colSpan: IDL set to 0]
     expected: FAIL
```

Comment:
It is suspicious that we don't pass all of these tests after the clamping changes. Have you looked into why that is?

Reply:
Yes, I took a brief look. It All of these failures seem to be related with how our attribute parsing code is handling values that are outside the limits of certain numerical types. For instance if you grep the metadata for more occurrences of 4294967296 and 2147483648 you will find other tests failing in the same way for other attributes. I suspect we aren't handling this case well for many attributes.